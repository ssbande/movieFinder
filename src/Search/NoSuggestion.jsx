import React from 'react';
import { Message } from 'semantic-ui-react';

const NoSuggestion = () => {
  return <Message size='tiny' color='yellow'>No data found ...</Message>
}

export default NoSuggestion;