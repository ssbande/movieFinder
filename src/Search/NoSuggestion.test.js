import React from 'react';
import { shallow } from 'enzyme';

import NoSuggestion from './NoSuggestion';

describe('No Suggestions', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<NoSuggestion />);
    expect(wrapper).toMatchSnapshot();
  })
})