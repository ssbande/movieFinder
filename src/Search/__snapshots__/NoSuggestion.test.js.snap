// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Suggestions should render correctly 1`] = `ShallowWrapper {}`;
